src/components/InfiniteList.js

```diff
@@ -4,11 +4,11 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { AutoSizer, List, InfiniteLoader } from 'react-virtualized'
-import styles from 'react-virtualized/styles.css'
+
+const noRowsHeight = 200
```

Comment:
Can we put this in the same utils file where we defined other UI elements breakpoint or heights?

Reply:
The Media queries breakpoints are stored in `_variables.scss`. I can't see which `.js` file you are mentioning in the `/utils` folder. Can you point it to me?

Reply:
Something like you did here https://github.com/clearlydefined/website/pull/484/files/beddda30a246928ae98c5dc959dd93eea3184d2a#diff-be73193a7de1c86c1903f5cf92321d2aR69

Reply:
I see. done